commands/ls.go

```diff
@@ -45,30 +45,22 @@ func runLsBug(cmd *cobra.Command, args []string) error {
 	allIds := backend.QueryBugs(query)
 
 	for _, id := range allIds {
-		b, err := backend.ResolveBug(id)
+		b, err := backend.ResolveBugExcerpt(id)
 		if err != nil {
 			return err
 		}
 
-		snapshot := b.Snapshot()
-
-		var author bug.Person
-
-		if len(snapshot.Comments) > 0 {
-			create := snapshot.Comments[0]
-			author = create.Author
-		}
-
 		// truncate + pad if needed
-		titleFmt := fmt.Sprintf("%-50.50s", snapshot.Title)
-		authorFmt := fmt.Sprintf("%-15.15s", author.DisplayName())
+		titleFmt := fmt.Sprintf("%-50.50s", b.Title)
+		authorFmt := fmt.Sprintf("%-15.15s", b.Author.Name)
 
-		fmt.Printf("%s %s\t%s\t%s\t%s\n",
-			colors.Cyan(b.HumanId()),
-			colors.Yellow(snapshot.Status),
+		fmt.Printf("%s %s\t%s\t%s\tC:%d L:%d\n",
+			colors.Cyan(b.Id),
+			colors.Yellow(b.Status),
 			titleFmt,
 			colors.Magenta(authorFmt),
-			snapshot.Summary(),
```

Comment:
That was the only usage of `Summary()` in the project and it was deprecated. You can safely remove it now.